release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -200,6 +201,13 @@ For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-co
 {product-title} 4.8 adds support for setting the thread count to increase the amount of incoming connections a cluster can handle.
 
 For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-setting-thread-count_configuring-ingress[Setting Ingress Controller thread count].
+=======
```

Comment:
Looks like this is a left-over from a merge conflict, which is messing up the rendered version.